### PR TITLE
Create `TaskDescription` message

### PR DESCRIPTION
**Pull request by dmytro-grankin**
This PR brings the Protobuf message for a task description. This change allows to avoid duplications of Spine validation options.

The manual checking of validity for task description changes was replaced by the validation rule `DescriptionChangeRule`.

Also was created `testdata.Given`, test utility for creating Protobuf messages, that allows to keep a test code more readable by avoiding multiple statements like this:

```java
TaskDescriptionVBuilder.newBuilder()
                       .setValue("smth")
                       .build()
```

### CODE REVIEW

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: testutil-api/src/main/java/io/spine/examples/todolist/testdata/MessageFactory.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import io.spine.examples.todolist.TaskDescription;
+import io.spine.examples.todolist.TaskDescriptionVBuilder;
+
+/**
+ * Utilities for creating Protobuf messages with a single field.
+ *
+ * @author Dmytro Grankin
+ */
+public class MessageFactory {
```
Comment (armiol) [inline testutil-api/src/main/java/io/spine/examples/todolist/testdata/MessageFactory.java]: Let's have a series of `Given...` instead.

File: api-java/src/main/java/io/spine/examples/todolist/c/aggregate/TaskPart.java [hunk 2]
```diff
@@ -138,20 +139,20 @@ public TaskPart(TaskAggregateRoot root) {
...
     @Assign
-    List<? extends Message> handle(UpdateTaskDescription cmd)
-            throws CannotUpdateTaskDescription, CannotUpdateTaskWithInappropriateDescription {
-        validateCommand(cmd);
-        final Task state = getState();
-        final StringChange change = cmd.getDescriptionChange();
+    List<? extends Message> handle(UpdateTaskDescription cmd) throws CannotUpdateTaskDescription {
+        boolean isValid = ensureNeitherCompletedNorDeleted(getState().getTaskStatus());
+        if (!isValid) {
+            throwCannotUpdateTaskDescription(cmd);
```
Comment (dmytro-grankin) [inline api-java/src/main/java/io/spine/examples/todolist/c/aggregate/TaskPart.java]: This is due to the Codecov bug.

File: testutil-api/src/main/java/io/spine/examples/todolist/testdata/MessageFactory.java (hunk 1 above)
Comment (dmytro-grankin) [inline testutil-api/src/main/java/io/spine/examples/todolist/testdata/MessageFactory.java]: Done.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: testutil/src/main/java/io/spine/examples/todolist/testdata/MessageFactory.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ */
+
+package io.spine.examples.todolist.testdata;
+
+import io.spine.examples.todolist.TaskDescription;
+
+/**
+ * Utilities for creating Protobuf messages.
+ */
+public class MessageFactory {
```
Comment (armiol) [inline testutil/src/main/java/io/spine/examples/todolist/testdata/MessageFactory.java]: This is a functional duplicate of `io.spine.examples.todolist.testdata.Given`. 

There can be only one.

Also, the `@author` tag is missing here.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

**Pull request merged**